visualization/dashboard/src/MLIDashboard/Cohort.ts

```diff
@@ -0,0 +1,128 @@
+import { IFilter, FilterMethods } from "./Interfaces/IFilter";
+import { JointDataset } from "./JointDataset";
+import { ModelExplanationUtils } from "./ModelExplanationUtils";
+
+export class Cohort {
+    public static CohortKey: string = "Cohort";
+    private static _cohortIndex: number = 0;
+
+    public rowCount: number = 0;
+    private readonly cohortIndex: number;
+    private mutateCount: number = 0;
+    private _filteredData: Array<{[key: string]: number}>;
+    private _cachedAverageImportance: number[];
```

Comment:
when is the underscore needed - I see these are private variables, but there are other private variables here without an underscore